views/includes/check-your-answers/individual-trustee.html

```diff
@@ -224,7 +224,7 @@
           "change-individual-beneficial-owner-still-involved-button"
         ) ]
       }
-    }), rows) %}
```

Comment:
Good fix, we should probably reorder this to check for no change first, following this pattern.

set shouldShowStillInvolved = (isTrustStillInvolved or isNotStillInvolved)

if not pageParams.isRegistration and not isNoChangeJourney
...
}) if shouldShowStillInvolved, rows) %}